src/Twilio/TwilioNoAuth.cs

```diff
@@ -0,0 +1,64 @@
+﻿using Twilio.Clients;
+using Twilio.Exceptions;
+using Twilio.Clients.NoAuth;
+
+namespace Twilio
+{
+    /// <summary>
+    /// Default Twilio Client
+    /// </summary>
+    public class TwilioNoAuthClient
+    {
+
+        private static string _region;
+        private static string _edge;
+        private static TwilioNoAuthRestClient _restClient;
+        private static string _logLevel;
+
+        private TwilioNoAuthClient() { }
+
+        /// <summary>
+        /// Get the rest client
+        /// </summary>
+        /// <returns>The rest client</returns>
+        public static TwilioNoAuthRestClient GetRestClient()
+        {
+            if (_restClient != null)
+            {
+                return _restClient;
+            }
+
+            _restClient = new TwilioNoAuthRestClient(region: _region, edge: _edge)
+            {
+                LogLevel = _logLevel
```

Comment:
`_logLevel` is not set in this file.
Make sure `_logLevel` is fetched from **TwilioOrgsTokenAuth**

Reply:
added function to set log level here, should we make these classes dependent on each other? is that design fine?